event_registration_partner_unique/readme/USAGE.rst

```diff
@@ -1,4 +1,8 @@
-#. Go to **Events > Events > Create**.
-#. Enable **Forbid duplicates**.
-#. Try to create 2 registrations for the same Attendee in this event.
-#. You can't.
+#. Go to **Events** and choose an event or create a new one.
+#. Enable **Forbid Duplicates**.
+#. Go to **Attendees**.
+#. Create a new attendee, starting with **Booked by**.
+#. Several attendees can have the same *Booked by* person or company, but if
+   you try to register an attendee by **Attendee Name** or by
```

Comment:
This isn't right.

Reply:
What is the problem?

Reply:
It won't raise on name duplication. Only if the attendee partner is the same. I already talked with @LudLaf and he is fixing it

Reply:
I thought the label of the field `attendee_partner_id` was that one. OK then.